job/job.go

```diff
@@ -65,7 +65,7 @@ func GenerateGenericJob(obj k8upv1alpha1.JobObject, config Config) (*batchv1.Job
 					RestartPolicy: corev1.RestartPolicyOnFailure,
 					Containers: []corev1.Container{
 						{
-							Name:      obj.GetMetaObject().GetName(),
+							Name:      string(obj.GetType()),
```

Comment:
Is there no String() method available?
```suggestion
							Name:      obj.GetType().String(),
```

Reply:
there is one 👍 looks like my editor thought otherwise, thanks 😊